talktorials/5_compound_clustering/T5_compound_clustering.ipynb

```diff
@@ -142,7 +142,7 @@
        "        "
```

Comment:
<p>Again, update plotting code (also renaming <code>dist_co</code> to <code>cutoff</code> to be in-line with the variable names before.</p><p> </p><p><code>cutoff = 0.2</code></p><p><code>clusters = ClusterFps(fingerprints,cutoff=cutoff)</code></p><p> </p><p><code># Plot the size of the clusters - save plot</code></p><p><code>fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(8, 2.5))</code></p><p><code>ax.plot(range(1, len(clusters)+1), [len(c) for c in clusters])</code></p><p><code>ax.set_xlim([1, len(clusters)])</code></p><p><code>ax.set_ylim([0, len(clusters[0])+1])</code></p><p><code>ax.set_xlabel('Cluster index', fontsize=20)</code></p><p><code>ax.set_ylabel('# molecules', fontsize=20)</code></p><p><code>ax.tick_params(labelsize=16)</code></p><p><code>plt.title('Threshold: '+str('%3.1f' %cutoff), fontsize=20)</code></p><p><code>plt.savefig("../data/T5/cluster_dist_cutoff_%4.2f.png" %cutoff, dpi=300, bbox_inches="tight", transparent=True)</code></p><p> </p><p><code>print('Number of clusters %d from %d molecules at distance cut-off %4.2f' %(len(clusters), len(mols), cutoff))</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>